### DISCUSSION TITLE
CustomEditor 인스펙터가 가능한가요?

### DISCUSSION THREAD

**Discussion by peterkangdev** [naverz/zepeto-studio-kor#941] [5. Q&A]
제목 그대로 CustomEditor 인스펙터가 가능한가요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요.
C#으로 만들어서 에디터 단에서만 사용하시는 것은 가능합니다.
하지만 제페토 타입스크립트로는 CustomEditor를 지원하지 않습니다. 
어떤 기능을 사용하고 싶으신지 보다 상세하게 제안을 해주시면
추후 로드맵에 반영할 수 있을 것 같습니다.
